_quarto.yml

```diff
@@ -232,49 +232,49 @@ website:
       collapse-level: 2
       align: left
       contents:
-        - section: "Essentials"
+        - section: "<span class='emoji-icon'>📌</span>  __Essentials__"
```

Comment:
Did you consider adding bold with CSS? This seems like a reasonable change that would be useful elsewhere, even if the icon isn't included.

Reply:
The `display: flex;` layout only works as intended if both the emoji and text are wrapped in an HTML element.